test/torchaudio_unittest/backend/soundfile/save_test.py

```diff
@@ -280,6 +281,7 @@ def test_fileobj_wav(self):
         self._test_fileobj('wav')
 
     @skipIfFormatNotSupported("FLAC")
+    @skipCUDAIfRocm
```

Comment:
The functionality tested here should be all Python. What kind of error do you see when you build with ROCm?

Reply:
> Thanks for the contribution.
> As I have no experience/knowledge in ROCm, I will leave the review of the ROCm part to those with expertise, but one naive question I got is the following.
> 
> At this moment, torchaudio only contains Python code and custom CPU code. There is no custom CUDA code. (although we plan to do add some CUDA kernel)
> I imagined that we could just build torchaudio with C++ complier and the corresponding PyTorch binary. Why does it need to ROCm at build time?


@mthrok, the reason was because I encountered multiple cmake errors from various modules failing to find the path to various rocm libraries.  I added the LoadHip cmake file and use it find all the various rocm libs.

Reply:
It was a mismatch error.

`==================================================================================== FAILURES ====================================================================================
________________________________________________________________________ TestFileObject.test_fileobj_flac ________________________________________________________________________

self = <torchaudio_unittest.backend.soundfile.save_test.TestFileObject testMethod=test_fileobj_flac>

    @skipIfFormatNotSupported("FLAC")
    @skipCUDAIfRocm
    def test_fileobj_flac(self):
        """Saving audio via file-like object works"""
>       self._test_fileobj('flac')

test/torchaudio_unittest/backend/soundfile/save_test.py:287: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
test/torchaudio_unittest/backend/soundfile/save_test.py:277: in _test_fileobj
    self.assertEqual(expected, found, atol=1e-4, rtol=1e-8)
/opt/conda/lib/python3.6/site-packages/torch/testing/_internal/common_utils.py:1285: in assertEqual
    exact_dtype=exact_dtype, exact_device=exact_device)
/opt/conda/lib/python3.6/site-packages/torch/testing/_internal/common_utils.py:1285: in assertEqual
    exact_dtype=exact_dtype, exact_device=exact_device)
/opt/conda/lib/python3.6/site-packages/torch/testing/_internal/common_utils.py:1296: in assertEqual
    super().assertTrue(result, msg=self._get_assert_msg(msg, debug_msg=debug_msg))
E   AssertionError: False is not true : Scalars failed to compare as equal! Comparing -0.99993896484375 and 1.00006103515625 gives a difference of 2.0, but the allowed difference with rtol=1e-08 and atol=0.0001 is only 0.00010001000061035157!
============================================================================ short test summary info =============================================================================
FAILED test/torchaudio_unittest/backend/soundfile/save_test.py::TestFileObject::test_fileobj_flac - AssertionError: False is not true : Scalars failed to compare as equal! Com...
=============================================================================== 1 failed in 1.07s ================================================================================`